src/rpc/core_rpc_server.cpp

```diff
@@ -1916,7 +1930,9 @@ namespace cryptonote
       std::list<std::pair<Blockchain::block_extended_info, std::vector<crypto::hash>>> chains = m_core.get_blockchain_storage().get_alternative_chains();
       for (const auto &i: chains)
       {
-        res.chains.push_back(COMMAND_RPC_GET_ALTERNATE_CHAINS::chain_info{epee::string_tools::pod_to_hex(get_block_hash(i.first.bl)), i.first.height, i.second.size(), i.first.cumulative_difficulty, {}, std::string()});
+        difficulty_type wdiff = i.first.cumulative_difficulty;
+        res.chains.push_back(COMMAND_RPC_GET_ALTERNATE_CHAINS::chain_info{epee::string_tools::pod_to_hex(get_block_hash(i.first.bl)), i.first.height, i.second.size(), 0, "", false, {}, std::string()});
```

Comment:
Why use `false` for the `uint64_t difficulty_top64` field? Seems unnecessarily confusing compared to just using 0.

Reply:
It used to be a uin64_t.